app/models/jsonld_record.rb

```diff
@@ -92,7 +93,8 @@ def title_language
   def metadata_map
     {
       author_display:        'creator',
-      description_display:   'description',
+      call_number_browse_s:  'call_number',
```

Comment:
I'm not totally in the loop here, but I think we want to use the `call_number_display` field instead. The `call_number_browse_s` field is used for sorting the call number browse list in Orangelight and doesn't include the full call number (it excludes the subfield $c). Here's an example where the values are different: https://bibdata.princeton.edu/bibliographic/3785401/solr